CHANGELOG.unreleased.md

```diff
@@ -29,7 +29,9 @@
 
 <!--packages-start-->
 
+- vhd-lib minor
+- xo-server minor
 - @vates/read-chunk major
-- xo-server patch
+- @xen-orchestra/backups minor
```

Comment:
I see you changed the order, what are you using to sort the lines?

`vscode` puts (like JS and the ASCII table) `@` before letters.